### PR TITLE
OpenStack: Fixups to BM workers documentation

### PR DESCRIPTION
**Pull request by mandre**
A couple of improvements to the openstack BM workers doc:
- Capitalize acronyms
- Fixed links
- Fixed typos
- Point to supported documentation for customizing OS image

### CODE REVIEW

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 1]
```diff
@@ -3,57 +3,57 @@
...
-    - By default, OpenStack networks support attaching both vms and bare-metal machines to them. 
+- Requirements:
+    - By default, OpenStack networks support attaching both VMs and bare-metal machines to them.
     * [Ironic bare metal service][2] can listen for, and PXE-boot machines in tenant networks
 
 - Create install-config.yaml:
 
     - Set compute.[worker].platform.openstack.type to the bare-metal server flavor.
-    - Set control-plane.platform.openstack.type to the vm flavor which will be used by the control-plane masters.
+    - Set control-plane.platform.openstack.type to the VM flavor which will be used by the control-plane nodes.
```
Comment (pierreprinetti) [inline docs/user/openstack/deploy_baremetal_workers.md:46]: While you're at it: I think it's `Control Plane` and not `control-plane`

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 2]
```diff
@@ -112,46 +110,45 @@ a preexisting network.
...
+## Deploy cluster with VM workers only and add BM workers after
+
+Initial cluster deployment is done with only
+VM workers attached to the installer-provisioned network. The bare-metal
 workers are added after initial installation. Bare-metal workers are
-attached to a pre-existing network. Traffic between masters and workers is routed between subnets.
+attached to a preexisting network. Traffic between masters and workers is routed between subnets.
+
+- Requirements:
+    - Cloud provider is configure to route traffic between VM subnets and bare-metal subnets.
```
Comment (pierreprinetti) [inline docs/user/openstack/deploy_baremetal_workers.md:137]: configured

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 3]
```diff
@@ -160,34 +157,34 @@ attached to a pre-existing network. Traffic between masters and workers is route
...
-        
-- wait for the installer to complete. 
+
+- wait for the installer to complete.
 
 - Once the installation is complete, enable routing between the subnet created by the installer and the preexisting subnet
-where the bare-metal machines will be attached. 
+where the bare-metal machines will be attached.
+
+- Deploy the bare-metal workers using `machinesets` and the [machine-api-operator][6]
```
Comment (pierreprinetti) [inline docs/user/openstack/deploy_baremetal_workers.md:176]: We could link to our spanking new MachineSet [upstream docs](https://github.com/openshift/installer/blob/master/docs/user/openstack/README.md#adding-a-machineset)

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 4]
```diff
@@ -112,46 +110,45 @@ a preexisting network.
...
+    - Cloud provider is configure to route traffic between VM subnets and bare-metal subnets.
 
-- Requirements: 
-    - Cloud provider is configure to route traffic between vm subnets and bare-metal subnets.   
-    
 - Create install-config.yaml:
-    - Set compute.[worker].platform.openstack.type to the vm flavor which will be used by worker nodes.
-    - Set control-plane.platform.openstack.type to the vm flavor which will be used by the control-plane masters.
-    
+    - Set compute.[worker].platform.openstack.type to the VM flavor which will be used by worker nodes.
```
Comment (pierreprinetti) [inline docs/user/openstack/deploy_baremetal_workers.md:140]: Please wrap `compute.[worker].platform.openstack.type` in backticks

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 5]
```diff
@@ -112,46 +110,45 @@ a preexisting network.
...
+    - Set control-plane.platform.openstack.type to the VM flavor which will be used by the control-plane nodes.
```
Comment (pierreprinetti) [inline docs/user/openstack/deploy_baremetal_workers.md:141]: Please wrap `control-plane.platform.openstack.type` in backticks

File: docs/user/openstack/deploy_baremetal_workers.md (hunk 3 above)
Comment (mandre) [reply docs/user/openstack/deploy_baremetal_workers.md:176]: Ah perfect, I was looking for that in MAO but couldn't find it.

File: docs/user/openstack/deploy_baremetal_workers.md (hunk 5 above)
Comment (pierreprinetti) [reply docs/user/openstack/deploy_baremetal_workers.md:141]: and `s/control-plane/controlPlane/`

**Pull request merged**